assets/src/application/ui/forms/ee-form/Submit.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+
+import { Button } from './adapters';
+import { SubmitProps } from './types';
+
+const Submit: React.FC<SubmitProps> = ({ submitting, label, ...rest }) => {
+	return (
+		<div className='submit-wrap'>
```

Comment:
maybe we should also have: 

- a reset button for setting form data back to initial values

- a cancel button for closing form and NOT saving data

Reply:
Yes, a reset button is already in the list but not sure what you mean by cancel button. The form will be submitted only when submit button is clicked. And why should we assume that the form will be rendered inside a modal to cancel it?

Reply:
ooo good point about the cancel button